endpointmanager/cmd/CHPLpopulator/main.go

```diff
@@ -122,33 +122,11 @@ func main() {
 		productNumber := chplEntry.CHPLProductNumber
 		productNumber = strings.TrimSpace(productNumber)
 
-		productName := chplEntry.Product.Name
-		productName = strings.TrimSpace(productName)
```

Comment:
did something change so we no longer have to trim space for these fields?

Reply:
Yes, so I had to change the way the CHPL endpoint list info file was created, which meant just adding the whole 'details' structure to the info file struct rather than pulling out each individual name entry which involved trimming the white spaces for these fields, but now that we are not pulling out the individual names this is no longer necessary. The individual fields are now parsed out in the parseHITProd function of the productquerier.go file, so not sure if I should add the trimspace there or not, it seems to be working as expected without the trimspace though, but I guess it couldn't hurt?.. what do you think?

Reply:
Ah okay, hmm... I mean if it's not a lot of effort to move the trimspace to the productquerier file, then we might as well just in case, I don't trust the data that's saved in CHPL lol